src/auth/bearer/fake/fake.cc

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2014, Treehouse Networks Ltd. New Zealand
```

Comment:
The author chose to contribute that particular file of code to Squid, under that license, for the reasons and purposes stated above. IMHO even having a clearly stated reason is irrelevant so long as the code is GPL-compatible and useful for Squid, as this code is